security_monkey/auditor.py

```diff
@@ -165,14 +167,17 @@ def _is_current_method_disabled(self):
 
         return False
 
-
     def audit_all_objects(self):
         """
         Read all items from the database and inspect them all.
         """
         self.items = self.read_previous_items()
         self.audit_these_objects(self.items)
 
+    def audit_specific_items(self, items):
```

Comment:
I have refactored this away. This did change logic in some other areas (refactored in latest push).  Some areas of the code need to `read_previous_items()` to get the items.

I have refactored this in the latest push where `auditor.items` must be set before running `auditor.audit_objects()`.